helix-rest/src/test/java/org/apache/helix/rest/server/TestAuthValidator.java

```diff
@@ -54,9 +55,18 @@ public class TestAuthValidator extends AbstractTestClass {
   private String _mockBaseUri;
   private CloseableHttpClient _httpClient;
 
+  private static String CLASSNAME_TEST_DEFAULT_AUTH = "testDefaultAuthValidator";
+  private static String CLASSNAME_TEST_CST_AUTH = "testCustomAuthValidator";
+
+  @AfterClass
+  public void afterClass() {
+    _gZkClient.deleteRecursively("/" + CLASSNAME_TEST_DEFAULT_AUTH);
+    _gZkClient.deleteRecursively("/" + CLASSNAME_TEST_CST_AUTH);
+  }
+
   @Test
   public void testDefaultAuthValidator() throws JsonProcessingException {
-    String testClusterName = "testDefaultAuthValidator";
+    String testClusterName = CLASSNAME_TEST_DEFAULT_AUTH;
```

Comment:
to be honest, you don't need another copy of the same string, directly use CLASSNAME_TEST_DEFAULT_AUTH..

Reply:
+1

Reply:
Updated.